### PR TITLE
Setup mobile version home page

### PR DESCRIPTION
**Pull request by bengigo**
Changes made in this branch:

- configure HTML&CSS linters
- create header and headline sections of the mobile version
- add style

### CODE REVIEW

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @bengigo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️

* For this project you need to add the 'works' and the 'about myself' sections, by looking at your code and design you seem like you didn't implement these sections, Kindly add them and then request another review 👍✔😀

* check the project requirements here : https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by bengigo** [comment]
Hello @Danieldotcomcoder 
I made a mistake and requested a review from the next project. I'm gonna request from the correct one but now I'm worried the next reviewer is not gone notice my mistake and take your review into consideration. Is there anything I can do to prevent that other than this message?

**Comment by Danieldotcomcoder** [comment]
Hello @bengigo, 😀👍 when you request the review you can send the reviewer a message explaining the mistake you made ,😀👍👍, and the next reviewer will check your project according to the  project requirements .

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <title>Bengi's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <header>
+            <nav>
+                <div id="logo">Bengi Genç</div>
```
Comment (aimemalaika) [inline index.html]: - [x] Kindly put the logo in an `a` tag instead of a div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                <img id="burger" src="images/burger-menu-icon.png" alt="burger menu icon">
```
Comment (aimemalaika) [inline index.html]: - [x] Kindly set the hamburger as a button inside the nav tag not an image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            </nav>
+        </header>
+
+        <section id="personal-intro">
+            <h1>I’m Bengi <br>Glad to see you!</h1>
+            <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and have a project you need coded, don’t
+                hestiate to contact me.</p>
+            <p class="connect">LET’S CONNECT</p>
```
Comment (aimemalaika) [inline index.html]: - [x] Kindly change this into a link or a button as requested in the project requirements

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @bengigo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Required Changes ♻️

- [ ] Check the comments bellow for required changes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @bengigo  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- you implemented all the changes requested by the previous reviewer 🥳 
- The UI looks familiar to the figma
- All linter checks are passing

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**